### PR TITLE
About page done

### PR DESCRIPTION
**Pull request by Hofkee**

### CODE REVIEW

File: ToNote/ViewModels/MainViewModel.cs [hunk 1]
```diff
@@ -72,5 +72,21 @@ public ICommand EditNoteCommand
...
+
+        public ICommand OpenAboutPageCommand
+        {
+            get
+            {
+                return _OpenAboutPageCommand ?? (_OpenAboutPageCommand = new RelayCommand(() =>
+                {
+                    var dialog = new AboutPageViewModel();
+
+                    var result = DialogService.OpenDialog(dialog);
```
Comment (t-eliks) [inline ToNote/ViewModels/MainViewModel.cs]: Just a nitpick, but if you're not doing anything with the returned value of a dialog, no need to store it in a variable.

File: ToNote/Views/MainView.xaml [hunk 2]
```diff
@@ -27,6 +27,10 @@
                 </ItemsControl.ItemsPanel>
             </ItemsControl>
         </ScrollViewer>
-        <Button Grid.Row="1" Content="Add note" HorizontalAlignment="Center" VerticalAlignment="Center" Padding="10" Margin="5" Command="{Binding AddNoteCommand}"/>
+        <StackPanel Orientation="Horizontal" Grid.Row="1">
+            <Button  Content="About" HorizontalAlignment="Center" VerticalAlignment="Center" Padding="10" Margin="5" Command="{Binding OpenAboutPageCommand}"/>
+            <Button  Content="Add note" HorizontalAlignment="Center" VerticalAlignment="Center" Padding="10" Margin="5" Command="{Binding AddNoteCommand}"/>
```
Comment (t-eliks) [inline ToNote/Views/MainView.xaml:32]: Attribute duplication on the buttons. It's about time we created some base styles so we don't need to go back later and clean up code.

File: ToNote/Views/AboutPageView.xaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        </Grid.RowDefinitions>
+
+        <StackPanel Grid.Row="0">
+            <TextBlock Text="Developers: " HorizontalAlignment="Center" Margin="0,10,0,0" DockPanel.Dock="Top" FontWeight="Bold" />
+        </StackPanel>
+        <Grid Grid.Row="1">
+            <Image Height="100" Width="100" HorizontalAlignment="Left" Margin="140,0,0,0" Stretch="UniformToFill" Source="https://media.npr.org/assets/img/2016/01/07/macaca_nigra_self-portrait_custom-a8e13582c9ca6f71f5cd62815b8bb5d6ff112dc2-s800-c85.jpg"/>
+            <TextBlock Text="Arijus Treinys" HorizontalAlignment="Left" Margin="140,0,00,0" DockPanel.Dock="Top" VerticalAlignment="Bottom" />
+            <Image Height="100" Width="100" HorizontalAlignment="Right" Margin="0,0,140,0" Stretch="UniformToFill" Source="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQrAzB3dynTfZ4CioA56_XksdHsXMZUZgv4HfSb5O9js5BBjEix" />
+            <TextBlock Text="Audrius Liuberskis" HorizontalAlignment="Right" Margin="00,00,140,0" DockPanel.Dock="Top" VerticalAlignment="Bottom" />
```
Comment (t-eliks) [inline ToNote/Views/AboutPageView.xaml]: Fairly certain that DockPanel.Dock doesn't do anything when the control is not in a dockpanel.

File: ToNote/ViewModels/MainViewModel.cs (hunk 1 above)
Comment (Hofkee) [reply ToNote/ViewModels/MainViewModel.cs]: Made some corrections

File: ToNote/Views/AboutPageView.xaml (hunk 3 above)
Comment (Hofkee) [reply ToNote/Views/AboutPageView.xaml]: Got it Sir!

File: ToNote/ViewModels/MainViewModel.cs [hunk 4]
```diff
@@ -72,5 +72,20 @@ public ICommand EditNoteCommand
...
+
+        public ICommand OpenAboutPageCommand
+        {
+            get
+            {
+                return _OpenAboutPageCommand ?? (_OpenAboutPageCommand = new RelayCommand(() =>
+                {
+                    var dialog = new AboutPageViewModel();
+                    DialogService.OpenDialog(dialog);
+
```
Comment (Kodnot) [inline ToNote/ViewModels/MainViewModel.cs:86]: [Nitpick] needless whitespace

**Review by Kodnot — APPROVED** [review]

**Comment by Hofkee** [comment]
[Add an about page](https://trello.com/c/gRvVKAVQ/14-add-an-about-page)

**Pull request merged**